src/cascadia/TerminalApp/lib/LeafPane.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+#include "Pane.h"
+#include "ParentPane.h"
+#include <winrt/TerminalApp.h>
+#include "../../cascadia/inc/cppwinrt_utils.h"
+#include <winrt/Microsoft.Terminal.TerminalControl.h>
+
+enum class Borders : int
+{
+    None = 0x0,
+    Top = 0x1,
+    Bottom = 0x2,
+    Left = 0x4,
+    Right = 0x8
+};
+DEFINE_ENUM_FLAG_OPERATORS(Borders);
+
+class LeafPane : public Pane, public std::enable_shared_from_this<LeafPane>
+{
+public:
+    struct SplitResult;

```

Comment:
I cannot find any other usage of "SplitResult" except this declaration.

Reply:
Yup, forgot to remove that